interface/system/update_channels.rst

```diff
@@ -3,31 +3,50 @@
 Software update
 ================
 
-.. _rockstor_licence:
+.. _rockstor_license:
 
-GNU GPLv2 Licenced
-------------------
+Licensed: FSF Free/Libre & OSI approved
+---------------------------------------
 
-The code that makes Rockstor possible is developed under the
-`GNU GPLv2 <https://www.gnu.org/licenses/old-licenses/gpl-2.0.html>`_ licence
-and is therefore fully Open Source, there is no contributor agreement and
+The Rockstor package code, as distributed, is developed under two main repositories:
+
+* Source0: `rockstor-core <https://github.com/rockstor/rockstor-core>`_ is
+  `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_ licensed.
+
+* Source1: `rockstor-jslibs <https://github.com/rockstor/rockstor-jslibs>`_ is
+  (`MIT <https://opensource.org/licenses/MIT>`_ AND
+  `Apache-2.0 <https://opensource.org/licenses/Apache-2.0>`_ AND
+  `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_ AND
+  `LGPL-3.0-or-later <https://www.gnu.org/licenses/lgpl-3.0-standalone.html>`_ AND
+  `ISC <https://spdx.org/licenses/ISC.html>`_) licensed.
+  Indicating the combined works in this jslibs repository.
+
+Making the package license, as per the `Fedora Project Wiki (Packaging:LicensingGuidelines)
+<https://fedoraproject.org/wiki/Packaging:LicensingGuidelines#Mixed_Source_Licensing_Scenario>`_:
+
+* *"GPL-3.0-or-later AND (MIT AND Apache-2.0 AND GPL-3.0-or-later AND LGPL-3.0-or-later AND ISC)"*
+
+See the `SPDX License List <https://spdx.org/licenses/>`_ for details on the above assertions.
+
+As such Rockstor the package is fully Open Source, there is no contributor agreement, and
```

Comment:
Minor style edits:
```suggestion
As such, Rockstor the package is fully Open Source; there is no contributor agreement, and
```

Reply:
@FroggyFlox Thanks, this does read better.